airflow-core/src/airflow/ui/src/i18n/locales/zh_TW/dashboard.json

```
@@ -0,0 +1,39 @@
+{
+  "welcome": "歡迎",
+  "stats": {
+    "stats": "統計",
+    "failedDags": "失敗的 DAGs",
+    "runningDags": "執行中的 DAGs",
+    "activeDags": "活躍的 DAGs",
+    "queuedDags": "排隊中的 DAGs"
+  },
+  "health": {
+    "health": "健康狀態",
+    "metaDatabase": "元資料庫",
```

According to https://zh.wikipedia.org/zh-tw/%E5%85%83%E6%95%B0%E6%8D%AE, it's probably better tranlated this way?

```suggestion
    "metaDatabase": "後設資料庫",
```

both translation doesn't really make sense to me though...

---

I think we should just invent a word for this. Ultimately this just means the place to store Airflow information, so maybe we can just call it 系統資料庫 or something. (Metadatabase is not a good name to begin with IMO but its usage is settled, but we can start fresh in other languages.)

---

sounds like an awesome idea!